index.js

```diff
@@ -0,0 +1,153 @@
+import { DateTime } from './modules/luxun.js';
+import ListOfBooks from './modules/books.js';
+
+const menu = document.querySelectorAll('.item-menu');
+const h1 = document.querySelector('h1');
+const contain = document.querySelector('.contain');
+const currentDate = document.querySelector('.current-date p');
+const listofbooks = new ListOfBooks();
+const error = document.getElementById('error');
+
+const menuActive = (menu, active) => {
+  for (let i = 0; i < menu.length; i += 1) {
+    menu[i].classList.remove('active');
+  }
+  active.classList.add('active');
+};
+const formatDate = () => {
+  const today = new Date();
+  const options = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+  };
+  return today.toLocaleDateString('en-US', options);
+};
+const generateContactForm = () => {
+  const parentElement = document.createElement('div');
```

Comment:
- [OPTIONAL] You have done great. Your project works great and is well organized, :+1: :+1: You have done a great job also dividing your code into modules. Your index.js file seems to have a lot of codes that could further be split into different modules. I'll suggest moving the functions and logic related to forms, `form validation, form creation, and form generation` into a new file inside the module. You could also move the functions for editing books into its module. :smile: :wave: :+1: